test/create.test.ts

```diff
@@ -217,11 +217,11 @@ describe("commands/create", function () {
     const privateKey = account.privateKey.slice(2);
     const consoleLog = spy(console, "log");
     const stdin = mockStd.stdin();
-    process.nextTick(() => {
+    setTimeout(() => {
```

Comment:
Why this change?

Reply:
Before, the entire program up to this point in the testing process was synchronous, which meant "nextTick" always meant the CLI was ready to accept input. Because of some asynchronous stuff that happens in the setup command, it no longer means it's ready at the next tic. The has to do with event loop ordering, I don't recall exactly how.

Reply:
cool, I buy it